ethanschreur/kids-krafts
# Customers are guided to send a contact form if they want their order shipped

**What does this PR do?**
- This pull request allows customers to request a shipped order

**Description of the task to be completed** 
- Add information about shipping to the "order details" page
- Press a link that takes them to the "contact us" page and fills out the message with details about their shipment request

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- If there are products on the shop page add them to your cart, otherwise add products with the status "selling" and go to http://127.0.0.1:5000/shop and then add them to your cart
- Go to http://127.0.0.1:5000/order_details
- View the shipping information and press the link that goes to the contact us form
- Confirm that the fields on the contact form are filled out with information about their cart
